pkgs/development/tools/rust/rustup/default.nix

```
@@ -128,7 +140,7 @@ rustPlatform.buildRustPackage rec {
     chmod +x $out/nix-support/ld-wrapper.sh
   '';
 
-  env = lib.optionalAttrs (pname == "rustup") {
+  env = lib.optionalAttrs (finalAttrs.pname == "rustup") {
```

One question tho: do we even need this condition? At the moment, the recipe is not generic and is `callPackage`d only once in the entire repo (and is never `overrideAttrs`ed either). Should I remove it at all?

---

@dianqk what's the point of that check? Why not just 
```suggestion
  env = {
```
?

---

Oops, I totally don't remember why I wrote it.

---

Okay, I've deleted it. `pname` isn't the kinda attr that gets overridden often, so no need to check against it here